test/models/api_test.rb

```diff
@@ -1727,9 +1755,9 @@ def test_getting_names
     assert_api_pass(params.merge(rank: "variety"))
     assert_api_results(names)
 
-    with    = Name.where.not("author is null or author = ''").
+    with    = Name.where.not(Name[:author].blank).
               reject(&:correct_spelling_id)
-    without = Name.where("author is null or author = ''").
+    without = Name.where(Name[:author].blank).
```

Comment:
Neat!  Is Model[:col].blank identical to the SQL "col is null or col = ''"??  Super useful!

Reply:
Yes. Those arel-extensions are so well written. `blank` does a `trim(coalesce(''))`